packages/functional-tests/src/app.ts

```diff
@@ -83,4 +85,56 @@ export default class App {
             delete this.activeTests[testName];
         }
     }
+
+
+    private async launchTestBrowser() {
+        while (true) {

```

Comment:
Sigh. There must be better solution than `while (true)`. Generators come to mind.

Reply:
I think it's fine actually.